### PR TITLE
Add precision and scale to decimal, clarify meaning of bytes

### PR DESCRIPTION
**Pull request by cpcloud**

### CODE REVIEW

**Comment by cpcloud** [comment]
cc @bkietz

**Review by jacques-n — COMMENTED** [review]
I wonder if we need to clarify the length of the bytes? Maybe we should state that is always 16 bytes?

**Comment by cpcloud** [comment]
Yeah, let's clarify the byte length.

File: binary/expression.proto [hunk 1]
```diff
@@ -59,6 +59,17 @@ message Expression {
             List list = 30;
         }
 
+        message Decimal {
+            // little-endian twos-complement integer representation of complete value (ignoring precision)
+            // Always 16 bytes in length
+            bytes value = 1;
+            // declared scale of decimal literal
+            int32 scale = 2;
```
Comment (jacques-n) [inline binary/expression.proto:67]: Minor nit here: do you mind putting precision first? We typically list it that way (and it helps me remember which is which, which I always forget).

**Review by jacques-n — APPROVED** [review]
Much obliged on the reorder!

**Pull request merged**